pkgs/applications/virtualization/virt-viewer/default.nix

```diff
@@ -22,8 +22,9 @@ stdenv.mkDerivation rec {
   buildInputs = [
     pkgconfig intltool glib libxml2 gtk3 gtkvnc gmp libgcrypt gnupg cyrus_sasl
     shared_mime_info libvirt libcap_ng yajl gsettings_desktop_schemas makeWrapper
-    xen numactl
-  ] ++ optionals spiceSupport [ spice_gtk spice_protocol libcap gdbm ];
+    numactl
+  ] ++ optionals spiceSupport [ spice_gtk spice_protocol libcap gdbm
+  ] ++ optional stdenv.is64bit xen;
```

Comment:
Reading ```xen.meta.platforms``` which is declared inside ```stdenv.mkDerivation``` is not possible on a platform not listed in ```xen.meta.platforms```, it triggers validation assert in ```stdenv.mkDerivation```.